accessibilityTest/Testing.ts

```diff
@@ -150,6 +150,30 @@ export const isInit = () => {
   return get(getRoot(), SearchInterface) != null;
 };
 
+export const afterQuerySelector = <T extends Element = Element>(parentNode: HTMLElement, selector: string) => {
+  return observeUntil(
+    parentNode,
+    {
+      childList: true,
+      subtree: true
+    },
+    () => parentNode.querySelector<T>(selector)
+  );
+};
```

Comment:
This is not a performant solution, but I used it because it's simpler to use and may be used again for future accessibility tests.

Would a faster solution be a better idea?

Reply:
These tests are not run often so don't need to be really fast. The time limit is 15 seconds here too or it will wait "forever"?